<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(98, 160, 234);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>551</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(81, 0, 135, 255), stop:0.427447 rgba(41, 61, 132, 235), stop:1 rgba(155, 79, 165, 255));</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="main_page">
     <widget class="QGroupBox" name="shape_group">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>120</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 190, 111);</string>
      </property>
      <property name="title">
       <string>   Choose shape</string>
      </property>
      <widget class="QRadioButton" name="triangle">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Triangle</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="trapezoid">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Trapezoid</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="circle">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Circle</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rectangle">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Rectangle</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="color_group">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>70</y>
        <width>120</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 97, 203);</string>
      </property>
      <property name="title">
       <string>   Choose color</string>
      </property>
      <widget class="QRadioButton" name="yellow">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Yellow</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="red">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Red</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="blue">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Blue</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="green">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Green</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="select">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>280</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="circle_page">
     <widget class="QGroupBox" name="groupBox_1">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>201</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(98, 160, 234);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>   Radius</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="r"/>
      </widget>
      <widget class="QSplitter" name="splitter_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>171</width>
         <height>71</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>     Area</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="c_area"/>
      </widget>
      <widget class="QSplitter" name="splitter_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>140</y>
         <width>171</width>
         <height>71</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>   Length</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="c_length"/>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="rectangle_page">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>126</width>
        <height>141</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(87, 227, 137);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QSplitter" name="splitter_25">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>106</width>
         <height>25</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>    b</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="r_b"/>
      </widget>
      <widget class="QSplitter" name="splitter_27">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>106</width>
         <height>25</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>    a</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="r_a"/>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>120</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(87, 227, 137);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QLCDNumber" name="r_p">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>90</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Perimeter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>      Area</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="r_area">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="triangle_page">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>126</width>
        <height>215</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 51, 59);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QSplitter" name="splitter_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>106</width>
         <height>25</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Side 2</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="side2"/>
      </widget>
      <widget class="QSplitter" name="splitter_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>106</width>
         <height>25</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Side 3</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="side3"/>
      </widget>
      <widget class="QSplitter" name="splitter_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>106</width>
         <height>25</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Side 1</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="side1"/>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>130</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 51, 59);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QLCDNumber" name="t_p">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>90</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_30">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Perimeter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_31">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>      Area</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="t_area">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="trapezoid_page">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>141</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(248, 228, 92);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QSplitter" name="splitter_22">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>106</width>
         <height>25</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>base 2</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="tr_b"/>
      </widget>
      <widget class="QSplitter" name="splitter_23">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>106</width>
         <height>25</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>   h</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="tr_h"/>
      </widget>
      <widget class="QSplitter" name="splitter_24">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>106</width>
         <height>25</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QSplitter" name="splitter_26">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>base 1</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="tr_a"/>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_28">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>106</width>
         <height>25</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_34">
        <property name="text">
         <string>side 1</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="tr_side_1"/>
      </widget>
      <widget class="QSplitter" name="splitter_29">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>106</width>
         <height>25</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_35">
        <property name="text">
         <string>side 2</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="tr_side_2"/>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>140</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(248, 228, 92);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QLCDNumber" name="tr_p">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>90</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_32">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Perimeter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_33">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>      Area</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="tr_area">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculate">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>555</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
